agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -366,14 +366,8 @@ bool backhaul_manager::socket_disconnected(Socket *sd)
 
             if (m_eFSMState >= EState::CONNECT_TO_MASTER) {
                 LOG(INFO) << "Sending topology notification on son_slave disconnect";
```

Comment:
I am not sure if the log print is used by tests for verification (aka Telemetry) - but if not should be changed to what is actually done here and move the print of sending topology notification inside.

Reply:
It is describing what actually being done. It just happens on the Topology Task event handler.
And it is a good thing to print it here because it helps to understand what is the source of the Topology notification.